hieradata/role/webserver.yaml

```diff
@@ -2,6 +2,8 @@
 psick::profiles::linux_classes:
```

Comment:
We need to remove linux_classes, this is windows based node

Reply:
In theory this could be used on both (the different params names for windows and linux and made to ease coexistence), but if the intention here is to have a webserver role only on windows, then it's ok to remove.

Reply:
Currently, yes. When we move to Linux, we'll update accordingly. Then again, there are lots of things that we can remove from the code. ;) I'll delete it.